app/controllers/completed_tasks_controller.rb

```diff
@@ -0,0 +1,36 @@
+class CompletedTasksController < ApplicationController
+  before_action :authenticate_user!
+  def new
+  end
+
+  def create
+  	if(params[:is_checked].to_s == "true")
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Use space after control keywords.<br>Tab detected.